### ISSUE TITLE
Wrong number of pods in a daemonset

### ISSUE THREAD

**Issue by lilithfr** [kubernetes/dashboard#2809]
##### Environment
```
Dashboard version: v1.8.2 
Kubernetes version: v1.9.2
Operating system: Debian 9
```

##### Observed result
In the Daemon Sets, the calico-etcd indicates that 1 pods out of 3 is running.
![screenshot from 2018-01-31 17-32-35](https://user-images.githubusercontent.com/5602250/35656664-59c2e718-06ad-11e8-9228-28233b0cfc9b.png)
But the kubectl indicates that only 1 pod is desired
```
$ kubectl get daemonsets -n kube-system
NAME          DESIRED   CURRENT   READY     UP-TO-DATE   AVAILABLE   NODE SELECTOR                     AGE
calico-etcd   1         1         1         1            1           node-role.kubernetes.io/master=   8d
calico-node   3         3         3         3            3           <none>                            8d
kube-proxy    3         3         3         3            3           <none>                            8d
```

##### Expected result
Except if I misunderstand something, the dashboard should indicates "1/1" and not "1/3" in the columns "Pods"

**Comment by floreks** [comment]
Can you also check the details page if it shows the wrong info?

**Comment by lilithfr** [comment]
Here de details page, I don't see anything special:
![screenshot from 2018-02-01 07-10-58](https://user-images.githubusercontent.com/5602250/35685660-287d41b8-071f-11e8-9eb6-cfb999c60b76.png)

**Comment by floreks** [comment]
Ok, so the details page shows pods status correctly but the list page not. We will check that.

**Comment by eloyekunle** [comment]
Fixed by #3131
/close
